src/main/ArrayVisualizer.java

```diff
@@ -483,19 +601,14 @@ public void run() {
                         if (ArrayVisualizer.this.updateVisualsForced > 10000) {
                             ArrayVisualizer.this.updateVisualsForced = 100;
                         }
-                    }
-                    catch (Exception e) {
+                    } catch (Exception e) {
                         e.printStackTrace();
                     }
-                    // See: https://stackoverflow.com/questions/580419/how-can-i-stop-a-java-while-loop-from-eating-50-of-my-cpu/583537#583537
-                    try {
-                        Thread.sleep(ArrayVisualizer.this.benchmarking ? 1000 : 0);
-                    } catch (InterruptedException e) {
-                        Thread.currentThread().interrupt();
-                        ArrayVisualizer.this.visualsEnabled = false;
-                    }
-
-                }}};
+                    long endTime = System.currentTimeMillis();
+                    statSnapshot.frameTimeMillis = endTime - startTime;
```

Comment:
think this should be averaged over time (like a second), it currently just jumps between a few simple numbers

Reply:
If you wanna implement it, go ahead. For me, it just sticks at 1000.0 consistently.